src/pages/index.js

```diff
@@ -19,32 +21,173 @@ const addCardButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#card-edit-modal");
 const addCardForm = addCardModal.querySelector(".modal__form");
 
+const avatarButton = document.querySelector(".profile__avatar-button");
+const avatarModal = document.querySelector("#avatar-edit-modal");
+const avatarForm = avatarModal.querySelector(".modal__form");
+
+const imagePopUp = new PopupWithImage({ popupSelector: "#preview-modal" });
+const popupDelete = new PopupDelete({ popupSelector: "#delete-card-modal" });
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c5770e84-6b43-47e8-8ef9-37352d6891bc",
+    "Content-Type": "application/json",
+  },
+});
+
+const userInfo = new UserInfo({
+  nameSelector: ".profile__name",
+  descriptionSelector: ".profile__description",
+  userAvatar: ".profile__avatar",
+});
+
+let cardSection;
+
+api.getInitialCards().then((initialCards) => {
+  cardSection = new Section(
+    {
+      items: initialCards,
+      renderer: (data) => {
+        const cardElement = new Card(
+          data,
+          "#card-template",
+          handleImageClick,
+          handleDelete,
+          handleLikeCard
+        );
+        cardSection.addItem(cardElement.getCard());
+      },
+    },
+    ".cards__list"
+  );
+  cardSection.renderItems();
+});
```

Comment:
Needs correction: don't forget to handle the api errors (add a `catch` block). I'm very sorry that didn't notice this

Reply:
No worries. Thank you.